feat(react-native): update @react-native-community/cli to 10.2.1 (#15738)
nrwl/nx: docs/shared/packages/react-native/react-native-plugin.md

To create additional React Native apps run:

```shell
nx g @nrwl/react-native:app <your-app-name>
```

### Generating Libraries

To generate a new library run:

```shell
nx g @nrwl/react-native:lib your-lib-name
```

### Generating Components

To generate a new component inside library run:

```shell
nx g @nrwl/react-native:component your-component-name --project=your-lib-name --export
```

Replace `your-lib-name` with the app's name as defined in your `tsconfig.base.json` file or the `name` property of your `package.json`

### Upgrade React Native

The Nx CLI provides the [`migrate` command](/core-features/automate-updating-dependencies) to help you stay up to date with the latest version of Nx.

#### Use upgrade-native Generator

To upgrade native iOS and Android code to latest, you can use the [upgrade-native](/packages/react-native/generators/upgrade-native) generator:

```shell
nx generate @nrwl/react-native:upgrade-native <your-app-name>
```

> This is a command that will replace the iOS and Android entirely.

Reviewer: Replace the iOS and Android what?

#### Upgrade Manually

> You can also upgrade React Native iOS and Android code use project [rn-diff-purge](https://react-native-community.github.io/upgrade-helper/).

Reviewer: ```suggestion
You can also upgrade React Native iOS and Android code using the [rn-diff-purge](https://react-native-community.github.io/upgrade-helper/) project.
```

Author: done.

### Start Metro Server

To start the server that communicates with connected devices:

```shell
nx start <your-app-name>
```

### Run iOS

> To build your app and starts it on iOS simulator or device:

Reviewer: ```suggestion
To build your app and start it on iOS simulator or device:
```

Author: changed

```shell
nx run-ios <your-app-name>
```

### Run Android

> To build your app and starts it on a connected Android emulator or device:

Reviewer: ```suggestion
To build your app and start it on a connected Android emulator or device:
```

Author: changed

```shell
nx run-android <your-app-name>
```

### Build iOS

> To builds iOS app:

Reviewer: ```suggestion
To build an iOS app:
```

Author: changed.

```shell
nx build-ios <your-app-name>
```

The build artifacts will be located under `<your app folder>/ios/build`.

> You can specify the build folder by passing option `buildFolder`:

Reviewer: ```suggestion
You can specify the build folder by setting the `buildFolder` option:
```

Author: changed.

```shell
nx build ios <your-app-name> --buildFolder="./build"
```

### Build Android

> To build Android app, run:

Reviewer: ```suggestion
To build an Android app, run:
```
